### ISSUE TITLE
graphic tablet draws without touching the tablet

### ISSUE THREAD

**Issue by Taza53** [xournalpp/xournalpp#3323]
**Affects versions :**
 - OS: [e.g. Linux (manjaro), not tested Mac, not tested Windows]
 - Version of Xournal++ [Xournal++ 1.1.0, libgtk: 3.24.30]

**Describe the bug**
When using a graphic tablet(one by wacom medium [037B]) I go between pages or go in a page, it starts to draw without touching the tablet.

**To Reproduce**
1. use tablet or mouse to take the cursor out of a page
2. use tablet to go in the page 
3. line will automatically start to draw.

**Input Logs**
Device:           Wacom One by Wacom M Pen
Kernel:           /dev/input/event12
Group:            4
Seat:             seat0, default
Size:             216x135mm
Capabilities:     tablet 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      disabled
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   none
Rotation:         n/a

**Expected behavior**
a line not to be drawn when going between pages while using a tablet

**Comment by LittleHuba** [comment]
Please invert the option "Merge button events with stylus tip events" under "Input System" in the preferences of Xournal++.

**Comment by Taza53** [comment]
Already using it.

**Comment by LittleHuba** [comment]
Let me rephrase this: If this option is checked, please uncheck it and if it is not checked, please check it. What you experience is most definitely caused by this option.

**Comment by Taza53** [comment]
yes the work around did work but the unchecked option is not for me, cause i find it easier to work with the option checked. Thanks for the help btw.

**Comment by Taza53** [comment]
Duplicate of #3274 https://github.com/xournalpp/xournalpp/issues/3274#issue-952855933
